jobbergate-cli/jobbergate_cli/subapps/job_scripts/app.py

```diff
@@ -309,3 +311,34 @@ def delete(
         "The job script was successfully deleted.",
         subject="Job script delete succeeded",
     )
+
+
+@app.command()
+@handle_abort
+def show_files(
+    ctx: typer.Context,
+    id: int = typer.Option(int, help="The specific id of the job script."),
+    plain: bool = typer.Option(False, help="Show the files in plain text."),
+):
+    """
+    Show the files for a single job script by id.
+    """
+    jg_ctx: JobbergateContext = ctx.obj
+    result = fetch_job_script_data(jg_ctx, id)
+
+    if jg_ctx.raw_output:
+        render_json(result.job_script_files)
+        return
+
+    main_file_path = str(result.job_script_files.main_file_path)
+
+    for (file_path, file_contents) in result.job_script_files.files.items():
+        if plain:
+            print()
+            print(f"# {file_path}")
+            if file_path == main_file_path:
+                print("# This is the main job script file")
+            print(file_contents)
```

Comment:
print statements

Reply:
Yes, these are here for the `--plain` option that should print the file contents in plain text to the terminal.